CotEditor/it.lproj/Localizable.strings

```diff
@@ -366,9 +366,9 @@
 "Hide indent guide lines" = "Nascondi le guide per l’indentazione";
 "Show indent guide lines" = "Mostra le guide per l’indentazione";
 
-"Keep on Top" = "Mantieni in alto";
+"Keep on Top" = "Mantieni in primo piano";
```

Comment:
Apple uses the same wording in Notes.app. Therefore, I suppose this line should not be changed.

<img width="670" alt="Screenshot 2023-04-19 alle 13 56 02" src="https://user-images.githubusercontent.com/1165044/232970938-50830212-b0f8-48df-8940-2fc73bca338f.png">

Reply:
The same issue exists in the Japanese localization. There are several different ways to localize "on top" in terms of windows. The data on the site is a bit older than the current system. For instance, the latest Notes uses the term "Keep on Top" but the data on the site is "Float on Top." With that also in mind, I, this time, decided to adopt the way in the latest Notes.app since it would be updated most recently.